src/templates/header.js

```diff
@@ -11,20 +12,95 @@ const links = [
   {title: 'Contact', link: '/contact'},
 ]
 
-const Header = ({ siteTitle }) => {
-  return (
-  <header>
-    <div>
-      <Nav 
-        logo={Logo({title: siteTitle})}
-        links={links}
-        GatsbyLink={Link}
-      />
-    </div>
-  </header>
-)
+class Header extends Component {
+
+  componentDidMount() {
+    if(typeof window !== 'undefined') {
+      let userDetails = {}
+      let SITE_NAME = window.location.origin
+      document.getElementById('contact-form') &&
```

Comment:
@Kameshwaran should we put this in JS only in the contact page. in header.js means it will come in every page right?

Reply:
Let's keep it in the header. 
This JS not only contains form handling logic, but also tracks the list of pages browsed by the visitor.  That info also sent in the form when a visitor submits it.

Reply:
@sanandnarayan

Reply:
@sanandnarayan